rr_platform/src/motor_relay_node/motor_relay_node.cpp

```diff
@@ -47,14 +54,32 @@ void SteeringCallback(const rr_platform::steering::ConstPtr &msg) {
 }
 
 void sendCommand(boost::asio::serial_port &port) {
-    std::string message = "$" + std::to_string(static_cast<char>(desiredSpeed)) + ", " +
-                          std::to_string(static_cast<char>(desiredSteer)) + "\n";
+    std::string message = "$" + std::to_string(desiredSpeed) + ", " +
+                          std::to_string(desiredSteer) + "," +
+                          std::to_string(p_const) + "," +
+                          std::to_string(i_const) + "," +
+                          std::to_string(d_const) + "," + "\n";
     
     try {
         boost::asio::write(port, boost::asio::buffer(message.c_str(), message.size()));
     } catch (boost::system::system_error &err) {
         ROS_ERROR("%s", err.what());
     }
+    ROS_INFO_STREAM("sent: " + message);
```

Comment:
We probably don't need to keep this kind of debugging printout around. If there are messages you frequently look back on to debug specific problems with the firmware, consider ROS_DEBUG_STREAM as an alternative which will expose the output to ROS (for reading via rqt_console or rosbag), but will, by default, hide the output from the terminal.
For more information on ROS logging levels and how to turn on debug output, see these pages:
http://wiki.ros.org/Verbosity%20Levels
http://wiki.ros.org/rosconsole#Configuration